cfgov/cal/views.py

```diff
@@ -43,6 +70,36 @@ def clean_filter_calendar(self):
         calendars = CFPBCalendar.objects.filter(title__in=calendar_names)
         return calendars
 
+class CalendarPDFForm(forms.Form):
+    filter_calendar = FilterCheckboxList(
+            choices = [(c.title, c.title) for c in CFPBCalendar.objects.all()],
+            required=True,
+            label='Calendar',
+            error_messages=ERROR_MESSAGES['CHECKBOX_ERRORS'])
+    filter_range_date_gte = FilterDateField(required=False,
+        error_messages=ERROR_MESSAGES['DATE_ERRORS'])
+    filter_range_date_lte = FilterDateField(required=False,
+        error_messages=ERROR_MESSAGES['DATE_ERRORS'])
+
+    def __init__(self, *args, **kwargs):
+        kwargs['error_class'] = FilterErrorList
+        super(CalendarPDFForm, self).__init__(*args, **kwargs)
+
+    def clean_filter_calendar(self):
+        calendar_names = self.cleaned_data['filter_calendar']
+        calendars = CFPBCalendar.objects.filter(title__in=calendar_names)
+        return calendars
+
+    def clean(self):
+        cleaned_data = super(CalendarPDFForm, self).clean()
+        from_date_empty = 'filter_range_date_gte' in cleaned_data and \
+                          cleaned_data['filter_range_date_gte']  == None
+        to_date_empty = 'filter_range_date_lte' in cleaned_data and \
+                        cleaned_data['filter_range_date_lte'] == None
+
+        if from_date_empty and to_date_empty :
+            raise forms.ValidationError(ERROR_MESSAGES['DATE_ERRORS']['one_required'])
+        return cleaned_data
```

Comment:
Entering invalid dates in the filter form (`CalendarFilterForm`) still raises an unhandled exception. Can/should this functionality be unified between the forms (through a common base class, for example), or is that outside the scope of this PR?